### DISCUSSION TITLE
the 'default email' field puts an Capital as first character of any given email address

### DISCUSSION THREAD

**Discussion by allesoverwaterstof** [JulianPrieber/littlelink-custom#337] [Issues]
As stated, the 'default email' field puts an Capital as first character of any given email address.

So when I make a default email link with custom title **t**est@mail.com
It shows as **T**est@mail.com on the webpage

**Comment by JulianPrieber** [comment]
This was a feature at one point, but this one might be redundant now, I'll look into it

**Reply by JulianPrieber** [reply]
This feature is indeed redundant and will be fixed in the next patch.

**Reply by allesoverwaterstof** [reply]
cheers
